src/components/ArticleTopics.jsx

```diff
@@ -0,0 +1,27 @@
+import { getAllTopics } from "../utils/api";
+import { useEffect, useState } from "react";
+import { Typography } from "@mui/material";
+import { Link } from "react-router-dom";
+
+export default function ArticleTopics() {
+  const [fetchedTopics, setFetchedTopics] = useState();
+
+  useEffect(() => {
+    getAllTopics().then((data) => {
+      setFetchedTopics(data);
+    });
+  }, []);
+  return (
+    <>
+      <Typography variant="h4">View Articles By Topic</Typography>
+      {fetchedTopics &&
+        fetchedTopics.map((topic, i) => (
+          <div key={i}>
```

Comment:
I think you can get rid of this div and move the key attribute down
![image](https://github.com/mohsinahmxd/nc-news/assets/151854815/0692fbcb-6444-4c3a-944d-cc65f36d9abb)